src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2611,6 +2611,39 @@ public bool RelativeMotionUnGrab()
             return ret;
         }
 
+        /// <summary>
+        /// It is designed to apply a blur effect to a window based on specified parameters. 
+        /// This function supports different types of blurring effects, 
+        /// including blurring the window's background only 
+        /// or blurring the area surrounding the window while keeping the window itself clear.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WindowBlurInfo BlurInfo
```

Comment:
다음과 같은 동작에서 Blur가 변경되나요?

``` c#
var blur = new WindowBlurInfo
{
     BlurType = Background,
     BlurRadius = 10,
};
Window.Instance.BlurInfo = blur;
blur.BlurRadius = 1;  // 이 set으로 인해서 Window의 blur가 변경되나요?
```
WindowBlurInfo가 Immutable아닐 이유가 있을까요?

Reply:
추가적으로 굳이 property로 만들지 않아도 되지 않나요? 메소드로 만들면 get을 구현할 필요가 없어서 더 간편해질 것 같습니다. 
```suggestion
        public void SetWindowBlurEffect(WindowBlurInfo blur);
```

Reply:
blur.BlurRadius = 1;  // 이 set으로 인해서 Window의 blur가 변경되나요?
(답변) 이것으로 인해 Window의 Blur 값이 변경되지는 않습니다. WindowBlurInfo는 DALi에 있는 data struct를 그대로 binding 한 것입니다.
WindowBlurInfo가 Immutable아닐 이유가 있을까요?
(답변) 단순 Data struct이므로 Immutable 아닙니다.
추가적으로 굳이 property로 만들지 않아도 되지 않나요? 메소드로 만들면 get을 구현할 필요가 없어서 더 간편해질 것 같습니다.
(답변) 네, 말씀하신 것처럼 Method로 제공하도록 하겠습니다.